src/LuaPrinter.ts

```diff
@@ -200,7 +200,7 @@ export class LuaPrinter {
             (luaLibImport === LuaLibImportKind.Require && luaLibFeatures.size > 0)
         ) {
             // Require lualib bundle
-            header += `require("lualib_bundle");\n`;
+            header += 'require("lualib_bundle");\n';
```

Comment:
If we are disallowing `` ` `` strings, we should at least be consistent, this string was changed to ' while the one on 194 was changed to "?

Reply:
Prettier uses non-default quotes type when the string contains default one, to avoid using escapes

> If we are disallowing ` strings

I'm not really sure about it myself, but I think being more consistent is better. If anything, I wouldn't mind switching to template strings everywhere ([quotes](https://eslint.org/docs/rules/quotes) has an option for that)